config/process_base.config

```
@@ -24,6 +24,9 @@ process{
   withLabel: mem_96 {
     memory = {check_memory(48.GB  + 48.GB * task.attempt, params.max_memory)}
   }
+  withLabel: max_mem {
```

For parallelism in the tags
```suggestion
  withLabel: mem_max {
```